CHANGELOG.md

```diff
@@ -255,7 +282,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Support for rsync & rclone replication
 - Helm chart to deploy operator
 
-[Unreleased]: https://github.com/backube/volsync/compare/release-0.8...HEAD
+[Unreleased]: https://github.com/backube/volsync/compare/release-0.8...release-0.9
```

Comment:
Is this really what you meant?

Reply:
hmm, yes - it'll need to be updated to the following once v0.9.0 releases:

```
[Unreleased]: https://github.com/backube/volsync/compare/release-0.9...HEAD
[0.9.0]: https://github.com/backube/volsync/compare/release-0.8...v0.9.0
```

so I kind of ended up in the middle - I guess I was considering the unreleased stuff (for the moment anyway until we add something that's not going into v0.9.0) are the items destined for v0.9.0.  Perhaps that is just confusing.

Reply:
I changed to compare to HEAD, think it does make the most sense ...